packages/qwik/src/qwikloader.ts

```diff
@@ -101,21 +114,27 @@ export const qwikLoader = (doc: Document, hasInitialized?: number) => {
             emitEvent('qerror', { importError: true, error, symbol, uri });
           }
         }
-        const previousCtx = (doc as any)[Q_CONTEXT];
+        if (!handler) {
```

Comment:
this wasn't handled before. this will only happen during skew version issue or connection problems

Reply:
sync qrls can never be wrong because they're part of the container and import error already emits an error, so you're just making the loader bigger.

You could move line 101/114 into here and break